PackageInfo.g

```diff
@@ -8,7 +8,7 @@ SetPackageInfo( rec(
 PackageName := "recog",
 Subtitle := "A collection of group recognition methods",
 Version := "1.3.2dev",
-Date := "15/04/2018", # dd/mm/yyyy format
+Date := "15/04/2019", # dd/mm/yyyy format
```

Comment:
Ouch, so 1.3.2 had a bad release date and I never noticed? :-(. Really need a check in ReleaseTools to check if the release date is in the past...